app/editor/src/features/content/list-view/constants/columns.tsx

```diff
@@ -8,6 +8,7 @@ export const columns: (Column<IContentModel> & UseSortByColumnOptions<IContentMo
     id: 'headline',
     Header: 'Headline',
     accessor: 'headline',
+    width: 300,
```

Comment:
Will these widths essentially ruin any smaller resolutions?

Reply:
It should be ok, without it it defaults to `150px`. That page does need to be made more responsive though. I was able to do it with the MediaTypes I am working on so I should be apply to apply it over there

![responsive](https://user-images.githubusercontent.com/15724124/166608667-1eec177c-4d5e-49dd-bbf7-45cfc4186fb4.gif)